mobile/lib/features/wallet/domain/wallet_history.dart

```diff
@@ -2,7 +2,14 @@ import 'package:get_10101/common/domain/model.dart';
 import 'payment_flow.dart';
 import 'package:get_10101/bridge_generated/bridge_definitions.dart' as rust;
 
-enum WalletHistoryItemDataType { lightning, onChain, trade, orderMatchingFee, jitChannelFee }
+enum WalletHistoryItemDataType {
+  lightning,
+  onChain,
+  trade,
+  orderMatchingFee,
+  jitChannelFee,
+  stable
```

Comment:
I can't remember if it's merged yet but at some point I made a new enum WalletType for this... By name, this enum should only really be used for transaction history types I think

Reply:
I don't mind if you refactor this further, it might be cleaner! I added it here as this is the one that's used currently in the `balance_row` . 
ideally, we should distinguish creating a stable position as a different history type!

Reply:
Ah yes, the it's in one of my PRs still :)